speid/__init__.py

```diff
@@ -74,14 +73,11 @@ def default(self, o):
 with open(STP_PRIVATE_LOCATION) as fp:
     private_key = fp.read()
 
-stpmex.configure(
-    wsdl_path=STP_WSDL,
+stpmex_client = Client(
     empresa=STP_EMPRESA,
     priv_key=private_key,
     priv_key_passphrase=STP_KEY_PASSPHRASE,
-    prefijo=int(STP_PREFIJO),
+    demo=SPEID_ENV != 'prod',
 )
```

Comment:
This should be moved to a different file and imported from there.

```python
from speid.processors import stpmex_client
```

for example